### PR TITLE
Android: build date fix

### PR DESCRIPTION
**Pull request by Thunder07**
Someone posted about this not working correctly on G+ community.

Not sure when this broke, but if i had to guess it was with the gradle upgrade.
I've added the build hour into the date as well, as it might be useful should you release multiple updates within the same day.

I got this fix from StackOverflow [post](http://stackoverflow.com/questions/3540739/how-to-programmatically-read-the-date-when-my-android-apk-was-built)

### CODE REVIEW

File: Source/ui_android/java/com/virtualapplications/play/MainActivity.java [hunk 1]
```diff
@@ -260,8 +243,8 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
 
 	private void displayAboutDialog()
 	{
-		long buildDate = getBuildDate(this);
-		String buildDateString = new SimpleDateFormat("yyyy/MM/dd", Locale.getDefault()).format(buildDate);
+		Date buildDate = BuildConfig.buildTime;
+		String buildDateString = new SimpleDateFormat("d MMM yyyy hh:mm a", Locale.getDefault()).format(buildDate);
```
Comment (jpd002) [inline Source/ui_android/java/com/virtualapplications/play/MainActivity.java]: Is there any reason behind the date format change? I think it should stay the same as before, if possible.

Comment (Thunder07) [inline Source/ui_android/java/com/virtualapplications/play/MainActivity.java]: the hours was meant to allow you to identify different release on the same day... or are you referring purely to the date format/structure?

Comment (jpd002) [inline Source/ui_android/java/com/virtualapplications/play/MainActivity.java]: Yeah, I was just referring to the date format. I think adding time is a good idea, so you can add it back if you want. Thanks!

Comment (jpd002) [inline Source/ui_android/java/com/virtualapplications/play/MainActivity.java]: Yeah, I was just referring to the date format. I think adding time is a good idea, so you can add it back if you want. Thanks!

Comment (Thunder07) [inline Source/ui_android/java/com/virtualapplications/play/MainActivity.java]: the american date format just doesn't click with me XD

**Comment by jpd002** [comment]
Awesome! Thanks a bunch!

**Pull request merged**